pallets/configuration/src/benchmarks.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+
+#![cfg(feature = "runtime-benchmarks")]
+
+//! Benchmarking
+use {
+    crate::{Call, Config, Pallet},
+    frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite},
+    frame_system::RawOrigin,
+    sp_std::vec,
+};
+
+benchmarks! {
+	set_config_with_u32 {}: set_max_collators(RawOrigin::Root, 100)
+
+	impl_benchmark_test_suite!(
+		Pallet,
+		crate::mock::new_test_ext(Default::default()),
+		crate::mock::Test
+	);
+}
+
+#[cfg(test)]
+mod tests {
+    use {super::*, crate::mock::Test, frame_support::assert_ok, sp_io::TestExternalities};
+
+    pub fn new_test_ext() -> TestExternalities {
+        let t = frame_system::GenesisConfig::default()
+            .build_storage::<Test>()
+            .unwrap();
+        TestExternalities::new(t)
+    }
+
+    #[test]
+    fn bench_register() {
+        new_test_ext().execute_with(|| {
```

Comment:
this looks like wrong ;)

Reply:
It was intentional to check if CI checks that, and it doesn't. It is supposed to check that these tests compile but not run them, right?

Reply:
We can evaluate later on if we should run tests with runtime-benchmarks

Reply:
```
running 10 tests
test mock::__construct_runtime_integrity_test::runtime_integrity_tests ... ok
test tests::config_sets_values_from_genesis ... ok
test tests::config_set_value ... ok
test benchmarks::bench_set_config_with_u32 ... ok
test tests::config_sets_default_values ... ok
test tests::config_set_many_values_different_sessions ... ok
test tests::config_set_many_values_same_block ... ok
test tests::config_set_many_values_different_blocks ... ok
test benchmarks::benchmark_tests::test_benchmarks ... ok
test benchmarks::tests::bench_set_config_with_u32 ... ok

test result: ok. 10 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
```

What's the difference between `benchmarks::bench_set_config_with_u32`, `benchmarks::benchmark_tests::test_benchmarks` and `benchmarks::tests::bench_set_config_with_u32`?